keel-actuator/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt

```diff
@@ -42,6 +44,12 @@ class ResourceActuator(
     if (!response.allowed) {
       log.debug("Skipping actuation for resource {} because it was vetoed: {}", id, response.message)
       publisher.publishEvent(ResourceCheckSkipped(apiVersion, kind, id))
+
+      if (resourceRepository.lastEvent(id) !is ResourceActuationPaused) {
+        log.info("Actuation for resource {} paused due to veto: {}", id, response.message)
```

Comment:
I think that we could just publish this event every time, and then set 
`IgnoreRepeatedHistory` on the event (used [here](https://github.com/spinnaker/keel/blob/master/keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlResourceRepository.kt#L149). That way, we don't have to always read the event history in this file. 

Also, the log message here is redundant - it says the same thing as the one on line 45.

Reply:
Oh, that’s useful! I didn’t know about it.

About the log message, it looks redundant, but since the check occurs on a recurring schedule and I’m only emitting the event the first time the veto kicks in after actuation had been in progress, I thought it was relevant to log. Of course, if we change this to always emit the event with the flag then it’s unnecessary. I’ll make that change.

Reply:
Added `ignoreRepeatedHistory = true` for `ResourceActuationPaused` and `ResourceActuationResumed`.